docs/textinput.md

```diff
@@ -995,6 +995,8 @@ The color of the `TextInput` underline.
 
 ### `value`
 
+> The `value` field implementation is broken in many subtle ways and is not recommended for use. Until these bugs are fixed, we recommend sticking with `defaultValue` and uncontrolled inputs.
```

Comment:
```suggestion
> The `value` field implementation is broken in many subtle ways and is not recommended for use. Until these bugs are fixed, we recommend sticking with `defaultValue` and uncontrolled inputs. See: [#4247](https://github.com/facebook/react-native-website/pull/4247)
```

Just something I usually like to see, some sort of issue where I can dig deeper.